rocky/reports/views/mixins.py

```diff
@@ -90,7 +90,7 @@ def save_report_data(
             report_id=uuid4(),
             organization_code=organization.code,
             organization_name=organization.name,
-            organization_tags=list(organization.tags.all()),
+            organization_tags=[tag.name for tag in organization.tags.all()],
```

Comment:
A helper function on the organization object that returns just the name would be useful, seeing that we already use this same loop in 4 places, and this will probably grow.